ijs-Run-TrackMate-Using-RoiManager-or-ResultsTable.groovy

```diff
@@ -0,0 +1,573 @@
+#@ Integer frameGap    ( label="Max Frame Gap [frames]" ) 
+#@ Double linkDistance ( label="Linking Max Distance [calibrated]" ) 
+#@ Double gapDistance  ( label="Gap Closing Max Distance [calibrated]" ) 
+#@ Boolean allowSplit  ( label="Allow Track Splitting" ) 
+#@ Double splitDistance ( label="Split Distance [calibrated]" )
+#@ RoiManager rm
+#@ ImagePlus imp
+
+/* 
+ *  Adapted from python: https://gist.github.com/ekatrukha/ac32d624665da81ad521ab4b040a4bad by Eugene Katrukha
+ *
+ * = CODE DESCRIPTION =
+ * Will run the TrackMate LAP tracker based on the ROIs in the ROI Manager instead of using TrackMate's spot detection results.
+ * It takes the centroid coordinates of the ROIs and uses the ROI area to estiamte a spot radius.
+ * It can also use the coordinates from a ResultsTable.
+ * 
+ * This script uses the LAP tracker from TrackMate and allows for splitting events. 
+ * This is then reflected in the names of the ROIs in the Roi Manager. 
+ * 	 Each cell will have a unique number when the track is created and that name will be prepended in the daughter cells.
+ *   Eg. After bing tracked for 6 frames, Cell 4 divides, 
+ *       the daughter cells will be called 4-1 and 4-2. 
+ *       If cell 4-2 divides, its daughter cells will be named 4-2-1 and 4-2-2
+ *   
+ *   Graph:         
+ *   ------
+ *   
+ *     4-1
+ *     |   
+ *  4 -      4-2-1
+ *     \    | 
+ *     4-2 -
+ *    		\
+ *    		 4-2-2
+ *    		 
+ * 
+ * == INPUTS ==
+ * An open image: Trackmate will keep the absolute path to this image in its settings file
```

Comment:
Maybe mention in the warning that only single channel images are supported

Reply:
written in line 46